.travis.yml

```diff
@@ -5,4 +5,4 @@ python:
   - "3.2"
   - "2.7"
   - "pypy"
-script: nosetests
+script: pytest
```

Comment:
We should update Testing in README and the requirements file.  That may fix Travis 3.2 build.